application/CohortManager/src/Functions/CohortDistributionServices/TransformDataService/TransformAction.cs

```
@@ -57,11 +58,10 @@ public override async ValueTask<object> Run(ActionContext context, RuleParameter
             }
             return participant;
         }
-        catch(Exception ex)
+        catch (Exception)
         {
-            throw;
+            return null;
```

This approach seems to suppress errors from this method, wouldn't we want re--throw this exception so it can be caught in the result of the rules?

---

exceptions are suppressed at some point in the rules engine, they do not come out of the other side. So returning null is the best option

---

I think the rules engine handles then and will map the error message in to the result.exceptionMessage field
It might be worth testing this though

---

it does not map it to the exception message, i checked